packages/gatsby/src/services/run-page-queries.ts

```diff
@@ -19,22 +19,23 @@ export async function runPageQueries({
   graphqlRunner,
 }: Partial<IQueryRunningContext>): Promise<void> {
   assertStore(store)
+  const state = store.getState()
 
   if (!queryIds) {
     return
   }
+
   const { pageQueryIds } = queryIds
-  const state = store.getState()
-  const pageQueryIdsCount = pageQueryIds.filter(id => state.pages.has(id))
-    .length
 
-  if (!pageQueryIdsCount) {
+  if (pageQueryIds.length === 0) {
     return
   }
 
   const activity = reporter.createProgress(
-    `run page queries`,
-    pageQueryIdsCount,
+    `run page queries ${
+      process.env.JEST_WORKER_ID && `(worker ${process.env.JEST_WORKER_ID})`
+    }`,
+    pageQueryIds.length,
```

Comment:
Should we drop activity text change from this PR? I think we would see `run page queries false` in regular mode with it as is

Reply:
we'd see `undefined`, yeah 🤔